docs/index.md

```diff
@@ -7,6 +7,7 @@ caption: Contents
 ---
 introduction.md
 resource-policy/index.rst
+memory/index.rst
```

Comment:
md?
```suggestion
memory/index.md
```

Reply:
Fixed. (Indeed it seems to work with index.md file. I copied this from the above line without really knowing what I'm doing. Perhaps the above line (resource-policy/index.rst) should be fixed, too. But that's another commit and we're going to work through the docs at some point anyway, so I ended up fixing only this.